src/engine/UI/menu/MenuOption.hpp

```diff
@@ -113,6 +113,7 @@ class InputOption : public MenuOption {
 
     virtual void OnHidden() override;
 
+    void SetOptionText(const std::string &text);
```

Comment:
This is very similar to `inline std::string GetInputText() const { return m_text; }`, perhaps we could move this to be next to `GetInputText` in the header and make this inlined or move `GetInputText` to also be defined in the .cpp file.

Also, choose a name for both that suits, either `Set/GetOptionText` or `Set/GetInputText`.

Reply:
Good call, I hadn't noticed `GetInputText` and agree that it fits better there